using-timescaledb/continuous-aggregates.md

```diff
@@ -2,29 +2,26 @@
 
 Aggregate queries which touch large swathes of time-series data can
 take a long time to compute because the system needs to scan large
-amounts of data on every query execution. TimescaleDB continuous
-aggregates automatically calculate the results of a query in the
-background and materialize the results. Queries to the continuous
-aggregate view are then significantly faster as they touch less raw
-data in the hypertable and instead mostly use the pre-computed
-aggregates to build the view.
-
-Continuous aggregates are somewhat similar to PostgreSQL [materialized
-views][postgres-materialized-views], but unlike a materialized view,
-continuous aggregates do not need to be refreshed manually; the view
-will be refreshed automatically in the background as new data is
-added, or old data is modified. Additionally, it does not need to
-re-calculate all of the data on every refresh. Only new and/or
-invalidated data will be calculated.  Since this re-aggregation is
-automatic, it doesn’t add any maintenance burden to your database.
-
+amounts of data on every query execution. To make such queries faster,
+continuous aggregates allows pre-computing (or materializing) the
+aggregates, while also providing means to continuously, and with low
+overhead, keep them up-to-date as the underlying source data changes.
+
+Continuous aggregates are somewhat similar to PostgreSQL's
+[materialized views][postgres-materialized-views], but, unlike a
+materialized view, a continuous aggregate can be continuously and
+incrementally refreshed, either manually or via a policy that runs in
+the background. A refresh can cover the entire aggregate or just a
+specific time range. In either case, the refresh only recomputes the
+aggregate buckets that have changed since the last refresh.
```

Comment:
That first sentence is really long and many nested phrases with commas between making it harder to read. I think you should split it up as it was in the original (you can replace the `;` with a `.`). Something like this:
```suggestion
Continuous aggregates are somewhat similar to PostgreSQL's
[materialized views][postgres-materialized-views] but, unlike a
materialized view, they do not need to be manually refreshed. A continuous aggregate can manually refreshed or refreshed incrementally using a policy that runs in
the background. A refresh can cover the entire aggregate or just a
specific time range. In either case, the refresh only recomputes the
aggregate buckets that have changed since the last refresh.
```